sdks/java/core/src/main/java/com/google/cloud/dataflow/sdk/util/BigQueryTableInserter.java

```diff
@@ -73,6 +79,23 @@
   // The initial backoff after a failure inserting rows into BigQuery.
   private static final long INITIAL_INSERT_BACKOFF_INTERVAL_MS = 200L;
 
+  // The maximum number of retry load jobs.
+  private static final int MAX_RETRY_LOAD_JOBS = 3;
+
+  // The maximum number of retries to poll the status of a load job.
+  private static final int MAX_LOAD_JOB_POLL_RETRIES = 10;
+
+  // The initial backoff for polling the status of a load job.
+  private static final int INITIAL_LOAD_JOB_POLL_BACKOFF_MILLIS = 60000;
```

Comment:
Use `TimeUnit` classes for clarity, something like:  `MILLISECONDS.convert(60, SECONDS)`